Makefile

```diff
@@ -31,11 +31,25 @@ help:
 	@echo "$$HELPTEXT"
 
 install:
-	poetry install
+	python -m venv $(VIRTUAL_ENV)
+	$(VIRTUAL_ENV)/bin/python -m pip install --upgrade pip setuptools wheel
+	$(VIRTUAL_ENV)/bin/python -m pip install --upgrade -rrequirements/test/requirements.txt
```

Comment:
Unix parameters frequently follow this pattern:

* Single-dash parameters _without arguments_ can be followed by other single dash parameters without spaces
  ```console
  $ sphinx-build -a -q -n -W -E -n docs/source docs/html
  $ sphinx-build -aqnWEn docs/source docs/html
  ```

* Single-dash parameters _with arguments_ immediately end argument interpretation; with or without intervening spaces, the next characters form the argument until the next space.
  ```console
  $ sphinx-build -aqnWEn -b linkcheck docs/source docs/html
  $ sphinx-build -aqnWEn -blinkcheck docs/source docs/html
  $ sphinx-build -aqnWEnblinkcheck docs/source docs/html
  ```

(You may have seen this in the past with C compiler invocations; it is very common to write options like `-funroll-loops`.)

This very specific case, involving the `-r` option for pip, is built into my muscle memory because early versions of tox [didn't allow `-r path/to/requirements.txt` (including a space)](https://github.com/tox-dev/tox/issues/658) and the only solution was to use `-rpath/to/requirements.txt` (no spaces).

This is why it works and why it entered into this PR; want it changed?

Reply:
No that's fine. This comment was more a request to learn than anything else.

Appreciate the explanation.